src/tests/ftest/dfuse/bash.py

```diff
@@ -60,52 +51,106 @@ def test_bashcmd(self):
+                            f"ls -a {abs_file_path1}",
+                            f"rm {abs_file_path1}",
+                            f"dd if=/dev/zero of={abs_file_path1} count={dd_count}"
+                            f" bs={dd_blocksize}",
+                            f"ls -al {abs_file_path1}",
+                            f"filesize=$(stat -c%s '{abs_file_path1}');"
+                            f"if (( filesize != {dd_count}*{dd_blocksize} )); then exit 1; fi",
+                            f"cp -r {abs_file_path1} {abs_file_path2}",
+                            f"diff {abs_file_path1} {abs_file_path2}",
+                            f"cmp --silent {abs_file_path1} {abs_file_path2}",
+                            f"rm {abs_file_path2}",
+                            f"mv {abs_file_path1} {abs_file_path2}",
+                            f"ls -al {abs_file_path2}",
+                            f"ls -al {abs_dir_path}/.",
+                            f"ls -al {abs_dir_path}/..",
+                            f"rm {abs_file_path2}",
+                            f"rmdir {abs_dir_path}",
+                            f"wc -l {fuse_root_dir}/src.c",
+                            f"xxd {fuse_root_dir}/src.c",
+                            f'ln -s src.c {link_name}',
+                            f'readlink {link_name}',
+                            f'realpath {link_name}',
+                            f'head {fuse_root_dir}/src.c',
+                            f'tail {fuse_root_dir}/src.c',
+                            # f'more {fuse_root_dir}/src.c', # more hangs over ssh somehow
+                            f'dos2unix {fuse_root_dir}/src.c',
+                            f"gcc -o {fuse_root_dir}/output {fuse_root_dir}/src.c",
+                            f'size {fuse_root_dir}/output',
+                            f'readelf -s {fuse_root_dir}/output',
+                            f'strip -s {fuse_root_dir}/output',
+                            f"g++ -o {fuse_root_dir}/output {fuse_root_dir}/src.c",
+                            f"gcc -c -o {fuse_root_dir}/obj_a.o {fuse_root_dir}/src_a.c",
+                            f"gcc -c -o {fuse_root_dir}/obj_b.o {fuse_root_dir}/src_b.c",
+                            f"ar -rc {fuse_root_dir}/lib.a {fuse_root_dir}/obj_a.o "
+                            f"{fuse_root_dir}/obj_b.o",
+                            f"objdump -d {fuse_root_dir}/obj_a.o",
+                            f"grep print {fuse_root_dir}/src.c",
+                            "awk '{print $1}' " + fuse_root_dir + '/src.c',
+                            f"base64 {fuse_root_dir}/src.c",
+                            f"md5sum {fuse_root_dir}/src.c",
+                            f"cksum {fuse_root_dir}/src.c",
+                            f"bzip2 -z {fuse_root_dir}/lib.a",
+                            f"chmod u-r {fuse_root_dir}/lib.a.bz2",
+                            f"curl \"https://www.google.com\" -o {fuse_root_dir}/download.html"]
```

Comment:
Can we make this `curl` something inside of our own network so that Internet connectivity failures don't cause this test to fail as it seems to have [here](https://jenkins-1.wolf.hpdd.intel.com:9090/job/daos-stack/job/daos/job/ci-daos-stack-argobots-PR-26-release-2.6/4/testReport/junit/FTEST_dfuse/DfuseBashCmd/Test___Functional_on_Leap_15_5___1___dfuse_bash_py_DfuseBashCmd_test_bashcmd_ioil_run_container_hosts_pool_server_config_engines_0_storage_0_5c37/)?

While I agree that this really ought not to be necessary, it clearly is.  :frowning: